tasktiger_admin/templates/tasktiger_admin/tasktiger.html

```diff
@@ -43,3 +51,7 @@ <h2>TaskTiger</h2>
 </table>
 
 {% endblock %}
+
+{% block tail %}
+  <script src="{{ admin_static.url(filename='admin/js/details_filter.js', v='1.0.0') }}"></script>
```

Comment:
Did you forget to add this file to git? Or how does this work?

Reply:
Hashed it out w/ Eric offline, apparently `details_filter.js` is a file in Flask-Admin: https://github.com/flask-admin/flask-admin/blob/master/flask_admin/static/admin/js/details_filter.js